generators/database-changelog-liquibase/templates/src/main/resources/config/liquibase/changelog/added_entity.xml.ejs

```diff
@@ -48,7 +48,7 @@
             <%_ for (relationship of relationships) {
                 if ((relationship.relationshipType === 'many-to-one' || (relationship.relationshipType === 'one-to-one' && relationship.ownerSide === true))
                             && !relationship.id) {
-                    relationship.otherEntity.primaryKey.ownFields.forEach(idField => {
+                    relationship.otherEntity.primaryKey.ids.map(id => id.field).forEach(idField => {
```

Comment:
seems like `ids.map(id => id.field)` is repeated everywhere, can we do this instead
when populating `relationship.otherEntity.primaryKey.ids` & `entity.primaryKey.ids` do a `map(id => id.field)` and collect that into `relationship.otherEntity.primaryKey.ownFields` and `entity.primaryKey.fields` so that these usages need not be touched and would be much nicer to read and use.

Reply:
In reality .ids should all be handled te same way, there should be no place where we handle ids that comme from a fields or a relationship. This is why I switched to ids.map(id => id.field) which is the original field in the entity it comes from.

But the current code contains distinctions, and If I want to have the least number of changes I have to sheet like that.
ownFields "represented" the id fields that come from the entity (it is why it was removed). Maybe we could have and `primaryKey.originalFields` which is easier to understand. But no matter what, this is not generalized enough to work with composite because you specifically instructed against that.

Reply:
What does it have to do with my comment. I don't want this map method repeated all over templates and would like to have it premapped on the field itself as seprate member thats it

Reply:
@deepu105 you asked to rename it back to `ownFields` I proposed to renameIt `originalFields` and explained wh the previous variable ownField should not exist and tried to provide more context. Will make the change right away.